services/comparison.py

```diff
@@ -481,6 +481,15 @@ def has_unintended_changes(self):
                 return True
         return False
 
+    def remove_unintended_changes(self):
+        filtered = []
+        for line in self._lines:
+            base_cov = line.coverage["base"]
+            head_cov = line.coverage["head"]
+            if (line.added or line.removed) or (base_cov == head_cov):
```

Comment:
This logic says if line is in the diff or coverage on the line has not changed, then it's unintended (coverage change on non-diff line) and we should filter it out. Correct?

Reply:
Correct yep